addr_linux.go

```diff
@@ -90,6 +90,10 @@ func AddrList(link Link, family int) ([]Addr, error) {
 			continue
 		}
 
+		if msg.Family != uint8(family) {
```

Comment:
I think this will cause passing in FAMILY_ALL to fail. You probably want: 'if family != FAMILY_ALL && msg.Family != uint8(family) {'